english/Find_the_cow.md

```diff
@@ -0,0 +1,3 @@
+Can you [find the cow ?] (https://github.com/udacity/create-your-own-adventure/blob/master/english/Find_the_cow.md)
```

Comment:
It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.